src/amo/components/AddonCompatibilityError/index.js

```diff
@@ -53,8 +54,12 @@ export class AddonCompatibilityErrorBase extends React.Component {
 
     if (reason === INCOMPATIBLE_NOT_FIREFOX) {
       message = i18n.sprintf(i18n.gettext(`You need to
-        <a href="%(downloadUrl)s">download Firefox</a> to install this add-on.`
+        <a href="%(downloadUrl)s">download Firefox</a> to install this
+        add-on.`
```

Comment:
was this line too long? Github makes it hard to tell. If it's not too long, it would probably better to put this back to where it was. Otherwise, our localizers will have a lot of busy work fixing fuzzy matches.